helm-chart/renku/values.yaml

```diff
@@ -460,6 +460,22 @@ ui:
     ## is available under https://5bcn9bqff4qt.statuspage.io/api/v2/summary.json, so
     # id: 5bcn9bqff4qt
 
+  ## Home settings allow configuration of the home page shown on RenkuLab when the user
+  ## is not logged in.
+  # homepage:
+  #   custom:
+  #     enabled: true
+  #     main:
+  #       contentMd: |
+  #         ## RenkuLab @ Yoyodyne
+  #         Welcome to the **Yoyodyne** RenkuLab instance!
+  #       backgroundImage:
+  #         url: https://eoimages.gsfc.nasa.gov/images/imagerecords/79000/79803/earth_night_rotate_lrg.jpg
+  #   tutorialLink: https://renku.readthedocs.io/en/latest/tutorials/01_firststeps.html
```

Comment:
Do we expect users to modify this? Can't it just be set automatically to https://renku.readthedocs.io/en/<renku-version>/tutorials/01_firststeps.html through chartpress?

Reply:
I personally would prefer to link to a tutorial that is a project on the RenkuLab instance, rather than link to a tutorial on an external site. In that case, it needs to be configurable.